src/apic-extension/azext_apic_extension/aaz/latest/apic/api/_update.py

```diff
@@ -86,15 +89,14 @@ def _build_arguments_schema(cls, *args, **kwargs):
         _args_schema.contacts = AAZListArg(
             options=["--contacts"],
             arg_group="Properties",
-            help="The contact information for the API.",
+            help="The set of contacts",
```

Comment:
The API spec provides different description message. Let me know if you want to change the messages back.